bin/check-package-metadata.js

```diff
@@ -44,7 +44,7 @@ async function checkMonorepoFile(packages, rootPath) {
   const monorepoFile = await fs.readFile('docs/site/MONOREPO.md', 'utf8');
 
   for (const p of packages) {
-    const packagePath = path.relative(rootPath, p.location);
+    const packagePath = path.relative(rootPath, p.location).replace('\\', '/');
```

Comment:
Please note that `.replace('\\', '/');` only replaces the first occurrence of `\`:

```
'x\\y\\zz'.replace('\\', '/') => 'x/y\\zz'
```

Reply:
Let's use `.replace(/\\/g, '/')` instead.